buffalo_poc/migrations/20180201154147_create_droplets.up.fizz

```diff
@@ -0,0 +1,6 @@
+create_table("droplets", func(t) {
+	t.Column("id", "uuid", {"primary": true})
+	t.Column("droplet", "string", {})
+	t.Column("droplets", "varchar[]", {})
+	t.Column("doToken", "string", {})
```

Comment:
Following the setup instructions leaves me with an install that fails because "dotoken" does not exist in the database. Looking at the examples provided for fizz, it looks like the preferred naming scheme?

I personally don't care what naming scheme we use unless it means fighting the naming scheme down the road.

Reply:
ya I was just trying to see if the paths generated accepted the struct variables by default.

Reply:
Gotcha.